<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>935</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>板形缺陷定量回归预测模型</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_9" stretch="2,4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="font">
           <font>
            <family>华光细圆_CNKI</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QGroupBox::title{
	font-size: 13px;
	font-weight: bold;
	}</string>
          </property>
          <property name="title">
           <string>板形回归预测</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>模型选择：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <item>
                <property name="text">
                 <string>CatBoost</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>BP</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>RNN</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>数据选择：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>选择数据</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>模型预测：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>开始预测</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7" stretch="2,1">
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="font">
             <font>
              <family>华光细圆_CNKI</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QGroupBox::title{
	font-size: 13px;
	font-weight: bold;
	}</string>
            </property>
            <property name="title">
             <string>预测结果</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="1" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                   <item>
                    <widget class="QComboBox" name="comboBox_2">
                     <property name="font">
                      <font>
                       <pointsize>12</pointsize>
                      </font>
                     </property>
                     <item>
                      <property name="text">
                       <string>选择系数</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>deg0</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>deg1</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>deg2</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>deg3</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>deg4</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pushButton_3">
                     <property name="font">
                      <font>
                       <pointsize>12</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>模型评估</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout">
                   <item>
                    <widget class="QLabel" name="label_4">
                     <property name="font">
                      <font>
                       <pointsize>12</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>R²</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="lineEdit">
                     <property name="font">
                      <font>
                       <pointsize>12</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_2">
                   <item>
                    <widget class="QLabel" name="label_7">
                     <property name="font">
                      <font>
                       <pointsize>12</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>MSE</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="lineEdit_2">
                     <property name="font">
                      <font>
                       <pointsize>12</pointsize>
                      </font>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QPushButton" name="pushButton_7">
                   <property name="font">
                    <font>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>图像显示</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_6">
                   <property name="font">
                    <font>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>保存单卷预测结果</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_4">
                   <property name="font">
                    <font>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>保存全部钢卷预测结果</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item row="0" column="0">
              <widget class="QComboBox" name="comboBox_3">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <item>
                <property name="text">
                 <string>选择想要评估的带钢号</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEdit">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QTextEdit" name="textEdit_2"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>935</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助文档</string>
    </property>
    <addaction name="help_doc"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="help_doc">
   <property name="text">
    <string>打开帮助文档</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
